policies/project-director-election-process.md

```diff
@@ -0,0 +1,76 @@
+# Project Director Election Process
+
+## Goals
+
+The purpose of this policy is to outline a clear and efficient process for electing new Project Directors to represent the Rust project on the board of the Rust Foundation. The process aims to achieve the following goals:
+
+* Select candidates who fulfill the outlined candidate criteria in the [role description document](TODO: link), focusing on new leadership and diverse perspectives rather than solely prioritizing the best candidate.
+    * Strive for tolerance and inclusivity by prioritizing selecting candidates that everyone can accept over selecting a subset of people's first choice.
+* Conduct the selection process in a time-efficient manner.
+* Incorporate input from a wide range of stakeholders across the entire Rust project.
+* Ensure a diverse pool of qualified candidates for selection.
+* Create a psychologically safe environment that values minority perspectives and encourages positive feedback, encouragement, and tolerance rather than preference or criticism.
+* Ensure participants feel positive about their interactions and the execution of the process.
+* Ensure the process is compliant with Rust Foundation bylaws and by extension Delaware corporation law.
+
+## Description
+
+### Setup
+
+* The council selects a facilitator by consent:
+    * Any team member can suggest a facilitator or volunteer themselves.
+    * The facilitator must not be a nominee in the election and must be someone everyone trusts to set aside personal preferences during the election process.
```

Comment:
It's not super clear to me who "everyone" here is and what we use as a signal. Maybe at least limiting to project members?

Reply:
I was assuming this meant "the council", though presumably if someone on a team objected they'd tell their council member and they'd be able to bring that input back to the rest of the council

Reply:
```suggestion
    * The facilitator must not be a nominee in the election and must be someone the council trusts to set aside personal preferences during the election process.
```

I think this makes me feel better -- in particular, unbounded sets of people on something nebulous like trust is always makes me nervous :)